tests/view_tests/tests/test_debug.py

```diff
@@ -238,14 +238,14 @@ def test_no_template_source_loaders(self):
     @override_settings(ROOT_URLCONF='view_tests.default_urls')
     def test_default_urlconf_template(self):
         """
-        Make sure that the default URLconf template is shown shown instead
+        Make sure that the default URLconf template is shown instead
```

Comment:
I merged this typo fix in d6d007819dd630bf844f17a64d5120f8d230a6c8.

Reply:
Cool, I’ll rebase on top of master now that #13463 is in as well.